.eslintignore

```diff
@@ -3,7 +3,8 @@
 bin/admin
 deploy
 migrations/archived
-public
+public/*
+!public/locales
```

Comment:
`!public/**/*.json` could work too, I see there's another json file that we might want to Prettify.

Not a blocker.